code/game/objects/items/weapons/clown_items.dm

```diff
@@ -238,43 +240,80 @@
 
 	if(isitem(target))
 		var/obj/item/target_item = target
+		if(target_item.current_glue_state != GLUE_STATE_NONE) //Check to see if its glued first.
+			to_chat(user,"<span class='warning'>It already has glue on it!</span>")
+			return
 		if(target_item.abstract) //Can't glue TK grabs, grabs, offhands!
 			return
 
-	to_chat(user,"<span class='info'>You gently apply the whole [src] to \the [target].</span>")
-	spent = 1
+	to_chat(user,"<span class='info'>You put some glue on \the [target].</span>")
+	if(uses < 0) //For superglue
+		spent = 1
+	uses--
 	update_icon()
 	apply_glue(target)
 
-/obj/proc/glue_act() //proc for when glue is used on something
+
+/obj/item/weapon/glue/temp_glue
+	name = "bottle of school glue"
+	desc = "An ordinary bottle of glue. Stickiness lasts for 3 minutes. <b>Non-toxic.</b>"
+	icon = 'icons/obj/items.dmi'
+	icon_state = "glue_safe"
+	w_class = W_CLASS_TINY
+	glue_duration = 3 MINUTES
+	uses = 4
+
+
+/obj/item/weapon/glue/temp_glue/update_icon()
+	if(uses)
```

Comment:
add `icon_state = glue_safe` here, just in case someone adds a way to refill your glue or a badmin busses it refilled, etc